terasoluna-gfw-parent/pom.xml

```diff
@@ -397,7 +398,7 @@
     <org.mybatis.version>3.4.6</org.mybatis.version>
     <org.mybatis.spring.version>1.3.2</org.mybatis.spring.version>
     <!-- == Dozer== -->
-    <dozer.version>6.2.0</dozer.version>
+    <dozer.version>6.3.0</dozer.version>
```

Comment:
- [ ] Dozer 6.4.0 has bean released. Please re-consider using version.

Reply:
OK,I'll upgrade Dozer's version to 6.4.0.Thank you.